ui/lib/replication/addon/templates/components/replication-table.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="vlt-table">
```

Comment:
Can we rename this to something that indicates it's specific to this card view?  maybe something like `replication-card-table`

Reply:
`vlt-table` is an existing style-only component, so i'm using it here intentionally, but i'll add a `replication` class to it as well so it's more specific. great suggestion!